silicon/README.md

```diff
@@ -0,0 +1,198 @@
+# Setup
```

Comment:
Can we move the setup instructions to the [platform specific instructions](https://rustc-dev-guide.rust-lang.org/building/prerequisites.html#platform-specific-instructions) section of the rustc-dev-guide? I'd really prefer not to have another top-level directory.

Reply:
These instructions will not be merged, that’s why I suggested you only review the single commit 😇

Reply:
Heh.